internal/provider/apikey_resource.go

```diff
@@ -126,6 +127,8 @@ func (r *apiKeyResource) Schema(ctx context.Context, _ resource.SchemaRequest, r
 			"disabled": schema.BoolAttribute{
 				Description: "Whether the API key is disabled.",
 				Optional:    true,
+				Computed:    true,
```

Comment:
is computed needed here?

Reply:
Yes, you can't set a default without also marking the field as computed. Default was necessary to prevent planned changes between an unset field and the response from Read. From [TF Docs](https://developer.hashicorp.com/terraform/plugin/framework/handling-data/attributes/bool#configurability) 

> [Optional](https://developer.hashicorp.com/terraform/plugin/framework/handling-data/attributes/bool#optional-1) and Computed: The value may be practitioner configured or the value may be set in provider logic when the practitioner configuration is null.

Reply:
TY